projects/canopy/src/lib/footer/footer.component.ts

```diff
@@ -10,6 +10,7 @@ import {
 interface Link {
   text: string;
   href: string;
+  id?: string;
```

Comment:
this might be a good opportunity to move this out into it's own interface file :-)

Reply:
as in the whole `Link`

Reply:
Good idea, done ✅